tools/ovc/openvino/tools/ovc/__init__.py

```diff
@@ -1,4 +1,19 @@
 # Copyright (C) 2018-2024 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
-from openvino.tools.ovc.convert import convert_model
\ No newline at end of file
+from openvino.tools.ovc.convert import convert_model
+from openvino.tools.ovc.telemetry_utils import is_optimum, init_mo_telemetry
+
+import importlib.metadata as importlib_metadata
+
+try:
+    optimum_version = importlib_metadata.version("optimum")
+except importlib_metadata.PackageNotFoundError:
+    optimum_version = None
+
+if is_optimum() and optimum_version is not None:
+    telemetry = init_mo_telemetry("Optimum Intel", optimum_version)
```

Comment:
the same for versions, optimum and optimum-intel, ti is 2 separated packages (the second one depends from the first) and both of them has own versions

Reply:
Do we need to collect both versions?

Reply:
let's focus on optimum-intel only for now, info about optimum version gives not much info for us (only which models are available for export)